specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionGatewaysListAll.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I was naming these based off the operation names. so connectionGatewaysListAll.json matched up with "ConnectionGate_ListAll" would you still like it renamed?

Reply:
fine to keep as is. I was thinking instead of connectionGatewaysList and connectionGatewaysListAll use connectionGatewaysListBySubscription and connectionGatewaysListByResourceGroup. (if you do rename, remember to rename for other resource types as well).